Cargo.toml

```diff
@@ -26,16 +26,16 @@ rustix = ["dep:rustix"]
 
 [dependencies]
 dyn-clone = "1.0.11"
-image = { version = ">=0.24", default-features = false, features = ["jpeg"] }
+image = { version = "^0", default-features = false, features = ["jpeg"] }
```

Comment:
`^0` is almost certainly not correct, `0.24.0` to `0.25.0` is a breaking change under Cargo semver requirements. Generally it's best to just specify full version numbers (setting the known compatible minimum patch version), relying on the default interpretation of `^`.